### PR TITLE
Update to apprentice items

### PR DESCRIPTION
**Pull request by Logintmg**
Resource requirement removed from apprentice items
Cast chance increased to 30% per lvl

### CODE REVIEW

**Comment by Logintmg** [comment]
I think section about other castable items requiring resource should be removed as well, maybe increasing silver amount.

**Comment by artyomtrityak** [comment]
Please put a PR towards `neworigins` branch, not `master`

File: gamedata.cpp [hunk 1]
```diff
@@ -2200,7 +2200,7 @@ ItemType id[] =
 	{"aegis","aegeis","AEGS",
 	 ItemType::DISABLED | ItemType::NOMARKET | ItemType::NOTRANSPORT,
 	 NULL,0,0,0, {{-1,0},{-1,0},{-1,0},{-1,0}},
-	 "CRAG",1,20, {{I_MITHRIL,1},{I_SILVER,600},{-1,0},{-1,0}},
+	 "CRAG",1,20, {{I_SILVER,2000},{-1,0},{-1,0},{-1,0}},
```
Comment (artyomtrityak) [inline gamedata.cpp:2203]: should we increase it further to like 5k? Aegis is pretty important item

File: gamedata.cpp [hunk 2]
```diff
@@ -2210,7 +2210,7 @@ ItemType id[] =
 	{"windchime","windchimes","WCHM",
 	 ItemType::DISABLED | ItemType::NOMARKET | ItemType::NOTRANSPORT,
 	 NULL,0,0,0, {{-1,0},{-1,0},{-1,0},{-1,0}},
-	 "CRWC",1,20, {{I_IRON,1},{I_SILVER,500},{-1,0},{-1,0}},
+	 "CRWC",1,30, {{I_SILVER,500},{-1,0},{-1,0},{-1,0}},
```
Comment (artyomtrityak) [inline gamedata.cpp:2213]: should we increase % chance to 40%? I feel like even 30% is too low for such items

**Comment by Logintmg** [comment]
made new pull request to a proper branch

**Pull request closed without merging**